common/web/types/src/kmx/kmx-plus/kmx-plus.ts

```diff
@@ -197,7 +192,7 @@ export class Strs extends Section {
   }
 
   /** process everything according to opts */
```

Comment:
```suggestion
  /** process everything according to opts, and add the string to this.allProcessedStrings */
```

"process everything according to opts" could use some wordsmithing to add meaning